.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: "TER deployment"
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  # The actual TER upload job
+  ter-release:
+    name: "TYPO3 TER release"
+
+    if: startsWith(github.ref, 'refs/tags/')
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: [7.4]
+
+    env:
+      TYPO3_EXTENSION_KEY: tea
+      TYPO3_USER: ${{ secrets.TYPO3_USER }}
+      TYPO3_PASSWORD: ${{ secrets.TYPO3_PASSWORD }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ${{ env.TYPO3_EXTENSION_KEY }}
+
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          extensions: intl, mbstring, xml, soap, zip, curl
+
+      - name: Install TYPO3 TER client
+        run: composer global require helhum/ter-client
+          "^0.1" --prefer-dist --no-progress
+
+      - name: Remove files
+        run: rm -rf CODE_OF_CONDUCT.md Tests codeception.yml phpcs.xml .github .gitlab .editor .Build .gitattributes .editorconfig .gitignore .php_cs.php Resources/Private/.eslintignore Resources/Private/.eslintrc.json Resources/Private/.prettierrc.js Resources/Private/package.json Resources/Private/stylelint.config.js
+
+      - name: List extension folder
+        run: ls -liAsh ./
+
+      - name: Upload EXT:${{ env.TYPO3_EXTENSION_KEY }}
+          as ${{ github.event.ref }} to TER
```

Comment:
I think we need to use the `>` operator for multi-line commands (or put it all in once line), e.g., like this:

https://github.com/oliverklee/ext-phpunit/blob/main/.github/workflows/ci.yml#L120

(I'm not completely sure that this is necessary, though).